translator/translate/otel/receiver/otlp/translator_test.go

```diff
@@ -94,6 +94,56 @@ func TestTracesTranslator(t *testing.T) {
 	}
 }
 
+func TestMetricsTranslator(t *testing.T) {
```

Comment:
Can we add a test case where the `otlp` section is an array since we support it?